origins_common/origins_common.module

```diff
@@ -85,3 +85,59 @@ function origins_common_form_entity_embed_dialog_alter(&$form, FormStateInterfac
   $form['attributes']['data-align']['#default_value'] = 'right';
   $form['attributes']['data-align']['#access'] = FALSE;
 }
+
+/**
+ * Implements hook_preprocess_table().
+ */
+function origins_taxonomy_access_preprocess_table(&$variables) {
+  // Add new 'Revision Number' column to the node revisions tab.
+  //
+  // Making this change in preprocess is not beautiful, but seems
+  // to be the only workable solution as the 'diff' module has
+  // already been enabled and has taken over the core route with
+  // its own controller. If we created our own controller to do
+  // this then we would have to call the contrib controller,
+  // which does not sound like a good idea.
+  if (\Drupal::routeMatch()->getRouteName() == 'entity.node.version_history') {
+    // Add 'Revision Number' column header.
+    $newhead = ['tag' => 'th', 'content' => ['#markup' => t('Revision Number')]];
+    // Insert this new column after the first one.
+    // Use array_splice as we don't know exactly how many columns there will be
+    // (and this code should work whether the 'diff' module has been enabled or
+    // not).
+    array_splice($variables['header'], 1, 0, [$newhead]);
+    if (isset($variables['header']['select_column_one'])) {
+      $variables['header']['select_column_one']['content'] = ['#markup' => t('Compare 1')];
+    }
+    if (isset($variables['header']['select_column_two'])) {
+      $variables['header']['select_column_two']['content'] = ['#markup' => t('Compare 2')];
+    }
+    // Loop through the rows.
```

Comment:
Irrelevant comment, we can see its looping though the rows. Pad this out to explain why you're looping the rows.

Reply:
Fixed